plugins/modules/proxmox_disk.py

```diff
@@ -254,8 +254,18 @@
   media:
     description:
       - The drive's media type.
+      - O(iso_image) is mandatory when O(media=cdrom) is used.
```

Comment:
This is a breaking change: before you could specify `media=cdrom` without having to specify `iso_image`.

Reply:
It seems so but before this PR proxmox_disk had no ability to work with CD-ROM and this option was absolutely useless. I spared 4 days trying to brute combination of options to create a CDROM and mount ISO. But no luck.
Then I recalled that I had once postponed implementation of CD-ROM manipulation for an undefined period. And implemented it. :)
Do you think that I should relax that requirement?

Reply:
Do you mean that the module always failed when `media=cdrom` was passed? Or is there a way (potentially using some other software / using a web UI) to use the module with `media=cdrom` before? In the former case this wouldn't be a breaking change, but in the latter case it would be. In the latter case it's better to relax the requirement.

Reply:
I can only say for myself - every try was failed. Proxmox needs special config line format for CD-ROM that includes `media=cdrom`.
I can make an assumption that someone else had successfully use this module for CD-ROM because there were no issues in tracker.
Anyway, it is not a problem to relax the requirement. Thank you for pointing out a potential breaking.